README.md

```diff
@@ -19,7 +19,7 @@ A micro-service for authentication.
 #### How to run
 See [Docker images](https://hub.docker.com/r/teamwanari/tutelar/tags)
 
-_TODO_
+docker build .
```

Comment:
I think pointing to the correct documentation would be better. I don't think if sb want to use this as a servis, he/she should clone the repo and build the container. Also I think the `how to run` part should point to configurations and run options and not to a simple "how to build the image" command.